cmd/cstor-volume-mgmt/controller/start-controller/controller.go

```diff
@@ -119,3 +121,15 @@ func getClusterConfig(kubeconfig string) (*rest.Config, error) {
 	}
 	return cfg, err
 }
+
+// getSyncInterval gets the resync interval from environment variable.
+// If missing or zero then default to DefaultSharedInformerInterval
+// otherwise return the obtained value
+func getSyncInterval() time.Duration {
+	resyncInterval, err := strconv.Atoi(os.Getenv("RESYNC_INTERVAL"))
+	if err != nil || resyncInterval == 0 {
+		glog.Warningf("Incorrect resync interval %q obtained from env, defaulting to %q seconds", resyncInterval, common.DefaultSharedInformerInterval)
+		return common.DefaultSharedInformerInterval
+	}
+	return time.Duration(resyncInterval) * time.Second
```

Comment:
time.ParseDuration would perhaps been a good choice.
@princerachit @kmova ?

> // ParseDuration parses a duration string.
// A duration string is a possibly signed sequence of
// decimal numbers, each with optional fraction and a unit suffix,
// such as "300ms", "-1.5h" or "2h45m".
// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".